rust/src/go/nmstate/Makefile

```diff
@@ -0,0 +1,13 @@
+# This Makefile is handling the go build, install, test and clean. It is needed
```

Comment:
Let's move this to go top level  `rust/src/go/Makefile`

Reply:
We are going to keep using `rust/src/go/nmstate/Makefile` as we discussed. Later we can use the vanity imports to simplify the import. Thanks!